### DISCUSSION TITLE
Linux Pipewire: Getting choppy audio

### DISCUSSION THREAD

**Discussion by Quackdoc** [LizardByte/community#481] [Sunshine] [answered]
I am dealing with getting choppy audio comming from pipewire and im not sure how to solve this and would like some help.

I have tried flatpak, appimage, aur sunshine-git and the official packages. All of them seem to have audio issues.

Arch Linux
Kernel 6.9.9
Pipewire 1.2.1-1

**Comment by Quackdoc** [comment, marked as answer]
I am a bit of an idiot, I had `PULSE_LATENCY_MSEC=60` which caused this, this is solved

**Reply by ReenigneArcher** [reply]
What did you change it to?
